scripts/download-overrides.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+
+import dnf.subject
+import hawkey
+import os
+import yaml
+import subprocess
+import sys
+
+
+# this was copied from coreos-koji-tagger
+def get_NVRA_from_NEVRA(string: str) -> str:
+    form = hawkey.FORM_NEVRA
+
+    # get a hawkey.Subject object for the string
+    subject = dnf.subject.Subject(string)  # returns hawkey.Subject
+
+    # get a list of hawkey.NEVRA objects that are the possibilities
+    nevras = subject.get_nevra_possibilities(forms=form)
+
+    # return the first hawkey.NEVRA item in the list of possibilities
+    rpminfo = nevras[0]
+
+    # come up with rpm NVRA
+    nvra = f"{rpminfo.name}-{rpminfo.version}-{rpminfo.release}.{rpminfo.arch}"
+    return nvra
+
+
+def is_override_lockfile(filename: str) -> bool:
+    return (filename.startswith('manifest-lock.overrides.')
+            and filename[-4:] in ['json', 'yaml'])
+
+
+assert os.path.isdir("builds"), "Missing builds/ dir; is this a cosa workdir?"
+
+rpms = set()
+for filename in os.listdir(os.path.join("src/config")):
+    if is_override_lockfile(filename):
+        with open(f'src/config/{filename}') as f:
+            lockfile = yaml.safe_load(f)
```

Comment:
just thought of this new piece:

on lins 31 above we support yaml or json, but here we are just doing `yaml.safe_load(f)`. We should handle both cases?

Reply:
Heh, here we're relying on the fact that JSON is a subset of YAML:

```
In [1]: import yaml

In [2]: with open('manifest-lock.overrides.x86_64.json') as f:
   ...:     j = yaml.safe_load(f)
   ...:

In [3]: j
Out[3]: {'packages': {'ignition': {'evra': '2.0.1-8.gita8f91fa.fc31.x86_64'}}}
```

Reply:
wow didn't realize that worked. Nice